spring-integration-core/src/main/java/org/springframework/integration/endpoint/AbstractPollingEndpoint.java

```diff
@@ -168,6 +169,9 @@ protected void onInit() {
 					this.taskExecutor = new ErrorHandlingTaskExecutor(this.taskExecutor, this.errorHandler);
 				}
 			}
+			if (this.transactionSynchronizationFactory == null) {
+				this.transactionSynchronizationFactory = new PassthroughTransactionSynchronizationFactory();
+			}
```

Comment:
We should probably take a look at other uses but I am not too concerned about, say, ImapIdle but, in this case, we would have added unnecessary overhead to a zillion pollers worldwide.

As I said, I don't mind if the test is for a non-empty advice chain, but if you want to further optimize the test to look for a tx interceptor, I am ok with that too. The only problem with that would be if someone wraps the tx interceptor for some reason.